### ISSUE TITLE
Use :focus-visible instead of :focus in buttons

### ISSUE THREAD

**Issue by ak4zh** [Brain-Bones/skeleton#236]
Adding the `:focus` pseudo-class to an element will make it show a focus specific styles and **disregard browsers heuristics**. The `:focus-visible`, in contrast, applies custom styling **only if it would be shown natively**. [Source](https://pawelgrzybek.com/the-difference-between-css-focus-and-focus-visible-pseudo-class/#css-focus-vs-focus-visible)

**More Reference:**
https://developer.mozilla.org/en-US/docs/Web/CSS/:focus-visible

**Comment by endigo9740** [comment]
@ak4zh After reviewing this I've made a small change to `core.css`. This will adjust the webkit touch tap style to be something more neutral (not hardcoded blue).

![Screen Shot 2022-09-30 at 4 50 32 PM](https://user-images.githubusercontent.com/1509726/193361114-3b79be88-fd18-4616-83fc-d1b65085dd43.png)

During my research I stumble across this, and I agree with their sentiment:
http://www.outlinenone.com/

For now I opted to leave the outline styles to their default state. We'll leave it to up to end users to decide what's best for them. My concern would be that certain combinations of theme colors might not provide good a11y.

If a11y is not a concern for you then please note that Tailwind provides a means to customize the outline styles:
https://tailwindcss.com/docs/outline-width

This includes:

- Width
- Color
- Style

Basically it works like the border styles do.

Thanks for bringing this up though!
